ui/app/controllers/vault/cluster/secrets/backend/sign.js

```diff
@@ -23,6 +23,12 @@ export default Controller.extend({
       }
     },
 
+    updateTtl(path, val) {
+      const model = this.get('model');
+      let valueToSet = val.enabled === true ? `${val.seconds}s` : undefined;
```

Comment:
can you have a value `undefined` for `valueToSet`, or does it need to default to something like 0 seconds?

Reply:
Really great question. As far as I can tell, it depends on the endpoint whether the API will accept and properly ignore an `undefined` value for ttl. Others will use the default if passed 0, but not all of them, so I've been defaulting to `undefined` for endpoints that allow it to be safe.

Reply:
Good to know.  Thanks for the answer.